MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h

```diff
@@ -101,6 +107,13 @@ static NSString *const kMSDistributionGroupIdKey = @"MSDistributionGroupId";
  */
 @property(nonatomic) id<MSAppDelegate> appDelegate;
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+@property(nullable, nonatomic) SFAuthenticationSession *authenticationSession;
+#pragma clang diagnostic pop
+#endif // __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
```

Comment:
This `#if` - `#endif` is not quite hard to read. Let's remove the comment at the end of the line.